### ISSUE TITLE
ko/docs/pytorch/956c67cb0f1c70fdb976dabf9f583b2950425b18/d3

### ISSUE THREAD

**Issue by Humae-JH** [runebookdev/runebook#613]
> Creates a categorical distribution parameterized by either <a href="#torch.distributions.categorical.Categorical.probs"><code>probs</code></a> or <a href="#torch.distributions.categorical.Categorical.logits"><code>logits</code></a> (but not both).

<a href="#torch.distributions.categorical.Categorical.probs"> <code>probs</code> </a> 또는 <a href="#torch.distributions.categorical.Categorical.logits"> <code>logits</code> </a>중 하나에 의해 파라미터 범주 형 분포를 작성  (두 파라미터를 동시에 이용하지 않음).
